### DISCUSSION TITLE
Direct coupling analysis error

### DISCUSSION THREAD

**Discussion by lewisr34** [prody/ProDy#1891] [Archived Issues]
I'm trying to run some multiple sequence alignment analyses with ProDy, but when I run the command "buildDirectInfoMatrix" I get an error that states "Python has stopped working.  A problem caused the program to stop working correctly.  Please close the program".  Is this a problem on my end, or a problem with the command?

I'm running Python 3.5.6 on a Windows machine in a Jupyter notebook.  I am able to successfully run other multiple sequence alignment analyses such as buildSCAMatrix.  Below is an example of my input:

msa = parseMSA('cytc.fasta')
msa_refine = refineMSA(msa, label=None, rowocc=None, seqid=None, colocc=1)
direct_info =buildDirectInfoMatrix(msa_refine, seqid = 0.8, psuedo_weight= 0.5)

--> This produces the error, and then my kernel in my Jupyter notebook dies and has to be restarted.

In contrast, this command works the way I want it to:
msa = parseMSA('cytc.fasta')
msa_refine = refineMSA(msa, label=None, rowocc=None, seqid=None, colocc=1)
SCA = buildSCAMatrix(msa_refine, turbo=True)

**Comment by jamesmkrieger** [comment]
It may be that the process is too memory intensive and the Jupyter notebook can't handle it. Could you try using a terminal instead and see if that works?

**Comment by lewisr34** [comment]
I ran it in a terminal, but I get the same error.  Do you know of any other ways to check if the error is caused by insufficient memory?

**Comment by jamesmkrieger** [comment]
You could maybe try running `top` or `htop` in another terminal and see how much memory it uses.

**Comment by jamesmkrieger** [comment]
Checking whether you ran out of memory isn't going to help you to not use too much memory. John (SHZ66) is going to look into the C/C++ code to fix it. He thinks he knows what the problem is and that it's related to using windows.

**Comment by jamesmkrieger** [comment]
@SHZ66, any news on this?
